hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -34,9 +31,19 @@ def get_supervised_imagenet_encoder() -> TileEncoder:
     return ImageNetEncoder(feature_extraction_model=resnet18, tile_size=224)
 
 
+def get_attention_pooling_layer(num_encoding: int = 512,
+                                pool_hidden_dim: int = 5,
+                                pool_out_dim: int = 1) -> Tuple[Type[nn.Module], int]:
+    pooling_layer = AttentionLayer(num_encoding,
+                                   pool_hidden_dim,
```

Comment:
If you're hard-coding the attention layer for these tests, might as well freeze the `pool_hidden_dim`, as the other combinations are tested somewhere else.

Reply:
I meant `pool_hidden_dim`, which doesn't affect the rest of the DeepMIL pipeline. I think `pool_out_dim` could still be useful to check.